prep-html-links/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Home Page</title>
+</head>
+<body>
+  <h1>Home page</h1>
+  <a href="file:///C:/Users/alank/repos/prep-0122-code-solutions/prep-html-links/contact-us/contact-us.html" target="">Click here to go to the contact page</a>
```

Comment:
Unfortunately this full file path would prevent anyone else from using your web pages. You'll want to use the second and fourth example file path styles in the W3Schools documentation.

![image](https://user-images.githubusercontent.com/7432943/148694011-349dc41e-aa73-4bc1-85be-655a627e85d8.png)